source/AndroidResolver/src/GradleResolver.cs

```diff
@@ -1161,7 +1161,7 @@ internal static bool SupportsAarFiles
             // To work around this when Gradle builds are enabled, explosion is enabled for all
```

Comment:
Update the comments for this?

Reply:
This comment is actually good with current code. It does not explicitly mention about "Gradle Project Export need to be enabled as well" part.

Reply:
Ah, sounds good